content/blog/pulumi-2-0-update/index.md

```diff
@@ -0,0 +1,76 @@
+
+There are a few core features still remaining to flesh out. The biggest one is retiring Dep in favor of Go modules everywhere. We are also adding support for aliases, transformations, and improved API projections, such as supporting union types. Finally, we'll be adding Go support for our Kubernetes, Docker, and Terraform remote state packages.
+
+## Policy as Code
+
+[CrossGuard, our Policy as Code framework]({{< relref "/docs/guides/crossguard" >}}), is now ready to take for a serious test drive. This feature enables you to write policies in real code and enforce them during updates. Policies can check for anything, however common checks include security, compliance, cost management, and general and team best practices. You can write your own policy packages or use off-the-shelf policy packs like our own AWSGuard package.
+
+CrossGuard is open source and the functionality is behind [the new `pulumi policy` command]({{< relref "/docs/reference/cli/pulumi_policy" >}}), as well as [the new `--policy-pack` flag]({{< relref "/docs/reference/cli/pulumi_up#options" >}}) for the `preview` and `up` commands. You can use this on any edition of Pulumi, without restriction, including the offline backends.
+
+If you choose to use CrossGuard with the Enterprise Edition of Pulumi, however, you'll get some added functionality. This includes server-side enforcement of policies and organizational policies, including "policy groups" which let you group and apply many policies together to your stacks, such as for instance applying different policies to production than development environments.
+
+![Policy Enforcement in the Console](./PolicyEnforcement.png)
+
+We recently added the ability to tag policy packs with a semantic version so that you can easily manage and apply packs using user-friendly versions instead of auto-generated numbers.
+
+The remaining areas of focus between now and launch include:
+
+* Enabling policy pack authoring in Python. Policies can already apply to stacks written in any language, no matter the language of the policy pack itself, but today policy packs are authored in Node.js. Python is coming soon and Go and .NET will follow afterward.
+
+* Allowing policy packs to be configured. This will let you selectively enable or disable specific rules, and/or set configuration variables on them, that influence their behavior, on a per stack or policy group basis.
+
+* Reference policy packs for AWS, Azure, GCP, and Kubernetes. These packs will be usable out-of-the-box with a collection of some of the most important configurable rules we've seen, in addition to being a starting point for creating your own packs.
+
+[CrossGuard is ready to try today]({{< relref "/docs/get-started/crossguard" >}}) and [we've begun blogging about some example use cases]({{< relref "/blog/manage-infrastructure-with-pac" >}}) that you may find interesting.
+
+## Other Goodies
```

Comment:
Do you also want to mention YAML export for k8s?

Reply:
Good idea! Will add.